opa/rego/rules/untrusted_checkout_exec.rego

```diff
@@ -32,21 +32,22 @@ build_github_actions[action] = {
 }[action]
 
 build_commands[cmd] = {
-	"npm": {"npm install", "npm run ", "yarn ", "npm ci(\\b|$)"},
-	"make": {"make "},
-	"terraform": {"terraform plan", "terraform apply"},
-	"gomplate": {"gomplate "},
-	"pre-commit": {"pre-commit run", "pre-commit install"},
+	"ant": {"^ant "},
+	"bundler": {"bundle install", "bundle exec "},
+	"cargo": {"cargo build", "cargo run"},
 	"go generate": {"go generate"},
-	"msbuild": {"msbuild "},
-	"maven": {"mvn ", "./mvnw "},
+	"gomplate": {"gomplate "},
 	"gradle": {"gradle ", "./gradlew "},
-	"bundler": {"bundle install", "bundle exec "},
-	"ant": {"^ant "},
+	"make": {"make "},
+	"maven": {"mvn ", "./mvnw "},
 	"mkdocs": {"mkdocs build"},
-	"vale": {"vale "},
+	"msbuild": {"msbuild "},
+	"npm": {"npm install", "npm run ", "yarn ", "npm ci(\\b|$)"},
 	"pip": {"pip install", "pipenv install", "pipenv run "},
-	"cargo": {"cargo build", "cargo run"},
+	"pre-commit": {"pre-commit run", "pre-commit install"},
+	"terraform": {"terraform plan", "terraform apply"},
+	"tofu": {"tofu plan", "tofu apply"},
```

Comment:
Should this be added under `terraform` like npm/yarn?

Reply:
either way is fine
feel free to submit to LOTP too - or at least put a placeholder Issue https://github.com/boostsecurityio/lotp/issues